ocaml/xapi/xapi_globs.ml

```diff
@@ -945,6 +947,8 @@ let repository_gpgkey_name = ref ""
 
 let repository_gpgcheck = ref true
 
+let bundle_dir = ref "/tmp/offline-update/bundle/"
```

Comment:
It's not a good idea to use fixed paths like this in `/tmp`. Paths in `/tmp` should be generated with a random component. Or this should go into `/var/xapi`

Reply:
When uploading a bundle file, Xapi will clear the content of bundle_dir created by the last uploading request. So if the path is a random path, it will not be cleared. So I move it to `/var/xapi`.